var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -44,7 +45,7 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
     variant("hiop", default=False, description="Enable/Disable HiOp")
     variant("ipopt", default=False, description="Enable/Disable IPOPT")
 
-    conflicts("~hiop~ipopt", msg="ExaGO needs at least one solver enabled")
+    conflicts("~hiop~ipopt @:1.4", msg="ExaGO needs at least one solver enabled")
```

Comment:
```suggestion
    conflicts("~hiop~ipopt @:1.4", msg="ExaGO needs at least one solver enabled. Only PFLOW is supported in 1.5+")
    # We will better support minimal builds with Python in future. You can use Python with PFLOW if desired ~ipopt~hiop
    conflicts("~hiop~ipopt+python", msg="ExaGO Python wrapper requires at least one solver enabled.")
```

Reply:
Presumably the text should be `PFLOW only mode is supported in 1.5+`

Reply:
Yes that would be fine. Idea is that the PFLOW application is the only one supported, as opposed to supporting PFLOW, OPFLOW and SCOPFLOW etc.

Reply:
updated